Package.resolved

```diff
@@ -10,6 +10,15 @@
           "version": "0.16.0"
         }
       },
+      {
+        "package": "CryptoSwift",
```

Comment:
Can you please revert this change?

We only add CryptoSwift as a dependency on Linux (or anywhere else `CommonCrypto` can't be imported).

Given that most SwiftLint developers use macOS, it's less noisy to leave this out.

Reply:
Okay, it showed up after running test, so I assumed it needed to be there.

Reply:
Yeah I think that's a bug in SwiftPM somewhere since CommonCrypto should always be importable on macOS.